docs/src/_includes/components/docs-toc.html

```diff
@@ -1,6 +1,6 @@
 <nav class="docs-toc c-toc" aria-labelledby="js-toc-label">
     {%- if all_content | toc | safe | length -%}
-    <h2 class="c-toc__label" id="js-toc-label">Table of Contents</h2>
+    <p class="c-toc__label" id="js-toc-label">Table of Contents</p>
```

Comment:
This is actually a subheading so I don’t think we should change the html tag here. You should be able to filter this out of the anchorjs results using :not(c-toc__label)